src/transform/HeaderTransform.css

```diff
@@ -0,0 +1,63 @@
+.pagelib_edit_section_link {
+  float: right;
+}
+
+/* todo: standardize on RTL selectors throughout the codebase. In many places we use .content-rtl. */
+html[dir='rtl'] .pagelib_edit_section_link,
+.content-rtl .pagelib_edit_section_link {
+  float: left;
```

Comment:
Related to the comment above 
https://github.com/wikimedia/wikimedia-page-library/pull/163/files#r22909979 - It's the edit transform's job to control default layout of edit links.

Reply:
This one is specific to the header since it needs to float. Scoped this one to .pagelib_header as well.